frontend/src/pages/GraphicalSelector/SidebarDrawer/styles.ts

```diff
@@ -0,0 +1,95 @@
+import styled, { css } from 'styled-components';
+
+const Wrapper = styled.div<{ open?: boolean }>`
+  position: absolute;
+  height: 85%;
+  width: 30rem;
+
+  top: 5rem;
+
+  background-color: white;
+  border-radius: 10px 0 0 10px;
+  filter: drop-shadow(-2px 2px 4px rgba(0, 0, 0, 0.3));
+
+  animation-duration: 0.7s;
+  animation-fill-mode: forwards;
+  ${({ open }) =>
+    open === undefined
+      ? css`
+          right: -30rem;
+        `
+      : open
+      ? css`
+          animation-name: animation_open;
+        `
+      : css`
+          animation-name: animation_closed;
+        `}
```

Comment:
understandable, but the nested ternarys are a tad hard to read hehe

Reply:
i agre